include/Core/UI/Setting/EmailLogSetting.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace EmailLog\Core\UI\Setting;
+use EmailLog\Core\UI\Setting\SettingField;
+
+class EmailLogSetting extends Setting {
+
+	/**
+	 * Implement `initialize()` method.
+	 */
+	protected function initialize() {
+		$this->section->id          = 'email-log';
+		$this->section->title       = __( 'Email Log Settings', 'email-log' );
+		$this->section->option_name = 'el_email_log';
+
+		$this->load();
+	}
+
+	/**
+	 * Implement `get_fields()` method.
+	 *
+	 * @return array
+	 */
+	public function get_fields() {
+		$fields = array();
+
+		$email_log_fields = array(
+			'allowed_user_roles' => __( 'Allowed User Roles', 'email-log' ),
+			'remove_email_logs'  => __( 'Remove Data on Uninstall?', 'email-log' ),
+		);
+
+		foreach ( $email_log_fields as $field_id => $label ) {
+			$field           = new SettingField();
+			$field->id       = $field_id;
+			$field->title    = $label;
+			$field->args     = array( 'id' => $field_id );
+			$field->callback = array( $this, 'render_email_log_settings' );
+
+			$fields[] = $field;
+		}
+
+		return $fields;
+	}
+
+	/**
+	 * Implement `render()` method.
+	 */
+	public function render() {
+	?>
+		<p><?php _e( 'Email Log Settings lets you control who can view Email Logs and lets you keep the Email Logs when you delete the plugin.', 'email-log' ); ?></p>
+	<?php
+	}
+
+	/**
+	 * Implement `sanitize()` method.
+	 *
+	 * @param mixed $values {@inheritDoc}
+	 *
+	 * @return mixed $values {@inheritDoc}
+	 */
+	public function sanitize( $values ) {
+		if ( ! is_array( $values ) ) {
+			return array();
+		}
+		foreach ( $values as $key => $value ) {
+			if ( $key === 'allowed_user_roles' ) {
+				$values[ $key ] = array_map( 'sanitize_text_field', $values[ $key ] );
+			} elseif ( $key === 'remove_email_logs' ) {
+				$values[ $key ] = sanitize_text_field( $value );
+			}
+		}
+		return $values;
+	}
+
+	/**
+	 * Renders the Email Log settings fields.
+	 *
+	 * @param array $args
+	 */
+	public function render_email_log_settings( $args ) {
```

Comment:
Split this up into two methods and assign them to each `SettingField`

Reply:
Fixed.

Reply:
Do you want me to isolate `sanitize()` function for each Setting field?

Reply:
For now, let's not change `sanitize`. But it needs to be changed eventually.